client/container/Market.jsx

```diff
@@ -0,0 +1,148 @@
+
+import Actions from '../core/Actions';
+import Component from '../core/Component';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import React from 'react';
+import HorizontalRule from '../component/HorizontalRule';
+import Table from '../component/Table';
+import BitShares from 'btsdex-fix';
+import CircularProgress from '@material-ui/core/CircularProgress';
+class Market extends Component {
+  static defaultProps = {
+    coin: {}
+  };
+
+static propTypes = {
+    coin: PropTypes.object.isRequired  
+};
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      cols: [
+        { key: 'price', title: 'BTC' },
+        { key: 'quantity', title: 'BWK' },      ],
+      orderbookasks:[],orderbookbids:[],orderdataret:[],loading : true,newrefresh: false
+    };
+  };
+
+  fetchOrderBook = async(update) => {
+    if(update){this.setState({newrefresh : true})}
+    try {
+      let quote = "BWK"
+      let base = "BTC"
+      quote =  `BRIDGE.${quote}`;
+      base = `BRIDGE.${base}`;
+    //Get orderbook data after connection to openledger WebSocket
+      BitShares.subscribe('connected', async () => {
```

Comment:
Can this bitshares code be moved into it's own file?

Reply:
Yes it can be, where do you suggest it be moved to?

Reply:
Let's make a new folder called client/utils/